wiring/inc/spark_wiring_print.h

```diff
@@ -66,6 +67,9 @@ class Print
     }
     virtual size_t write(const uint8_t *buffer, size_t size);
 
+    virtual size_t write(uint8_t b, system_tick_t timeout) { return write(b); }
```

Comment:
I would prefer to be careful here and add the new `write` methods to TCPClient only. The reason is that there is only one stream implementation that supports it, and I imagine users will be confused with the presence of the new methods that don't do anything useful for most streams.